tests/framework/installer/test_data.go

```diff
@@ -0,0 +1,87 @@
+package installer
+
+import (
+	"github.com/rook/rook/tests/framework/utils"
+)
+
+func GetBlockPoolAndStorageClassDef(poolName string, storageClassName string, namespace string) string {
+	return `apiVersion: rook.io/v1alpha1
+kind: Pool
```

Comment:
to remove the rest of the duplication, i'd suggest you have one helper for the pool, one for the storage class, and one for the pvc. Then you can have a helper that concatenates them.

Reply:
Done, split them into one helper each for pool,storageclass and PVC. However i have one helper that does all 3 combined, becuase we wanted to have coverage for issue #767